app/dashboard/templates/dashboard/recovery_code.html

```diff
@@ -9,8 +9,8 @@
     <div class="card-body">
       <h1 class="h3">Recovery codes</h1>
       <p>
-        If you lose access to your authentication device, you can use one of these backup codes to login to your
-        account. Each code may be used only once. Make a copy of these codes, and store it somewhere safe.
+        In the event you lose access to your authenticator app you can use one of these recovery codes to gain access to your account again.
+        Each code may be used only once and you will not be able to view these again, make sure to store them in a safe place.
```

Comment:
The current implementation allows user to see the recovery codes again :). This is debatable as some services show the recovery codes only once while others allow user to access the codes again ...